code/modules/clothing/suits/wintercoats.dm

```diff
@@ -181,8 +181,9 @@
 	armor = list(MELEE = 25, BULLET = 15, LASER = 30, ENERGY = 40, BOMB = 25, BIO = 0, FIRE = 0, ACID = 45)
 
 // Head of Security
-/obj/item/clothing/suit/hooded/wintercoat/security/hos
+/obj/item/clothing/suit/armor/hos/trenchcoat/winter
 	name = "head of security's winter coat"
+<<<<<<< HEAD
```

Comment:
Cant do large comments from mobile but we can scrap all the HEAD, was just the overwritten descriptions that I should have done modularly anyways